packages/trader/src/AppV2/Components/PurchaseButton/purchase-button-content.tsx

```diff
@@ -6,55 +6,35 @@ import { getLocalizedBasis } from '@deriv/shared';
 import { Money } from '@deriv/components';
 
 type TPurchaseButtonContent = {
-    current_stake?: number | null;
     error?: React.ReactNode;
+    has_no_button_content?: boolean;
     info: ReturnType<typeof useTraderStore>['proposal_info'][0] | Record<string, never>;
     is_reverse?: boolean;
-    is_high_low?: boolean;
 } & Pick<
     ReturnType<typeof useTraderStore>,
-    | 'currency'
-    | 'has_open_accu_contract'
-    | 'is_accumulator'
-    | 'is_multiplier'
-    | 'is_vanilla_fx'
-    | 'is_vanilla'
-    | 'is_turbos'
-    | 'is_touch'
+    'currency' | 'has_open_accu_contract' | 'is_multiplier' | 'is_vanilla' | 'is_turbos'
 >;
 
 const PurchaseButtonContent = ({
     currency,
-    current_stake,
     error,
     has_open_accu_contract,
+    has_no_button_content,
     info,
-    is_accumulator,
-    is_high_low,
     is_multiplier,
     is_turbos,
     is_vanilla,
-    is_vanilla_fx,
     is_reverse,
-    is_touch,
 }: TPurchaseButtonContent) => {
-    const { current_stake: localized_current_stake, payout, stake } = getLocalizedBasis();
+    if (has_no_button_content && !error) return null;
+
+    const { payout, stake } = getLocalizedBasis();
 
     const getAmount = () => {
         const { stake, obj_contract_basis } = info;
-
-        if (is_multiplier) return stake;
-        if (is_accumulator) return Number(current_stake);
```

Comment:
Removed `is_accumulator` from checks in this component as we do not render content for Accumulator purchase button anymore.